2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,21 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+friends
+  .filter((friend) => friend.age > 34)
+  .forEach((friend) => {
+    thirtyFiveOrOlder.push(friend);
+  });
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
+friends
+  .filter((friend) => friend.company === "POWERNET")
+  .forEach((friend) => powerNetEmails.push(friend.email));
```

Comment:
Same as above, you can use a filter to apply the condition, and then a map to extract the fields that you need to return.